### PR TITLE
Add lookupApplicantSync to ApplicantRepository

### PR DESCRIPTION
**Pull request by natsid**
Helpful for tests that want to compare before/after state of an applicant that would otherwise just immediately `join()` the async call.

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/app/repository/ApplicantRepository.java [hunk 1]
```diff
@@ -41,4 +41,8 @@ public ApplicantRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exe
         },
         executionContext);
   }
+
+  public Optional<Applicant> lookupApplicantSync(long id) {
+    return ebeanServer.find(Applicant.class).setId(id).findOneOrEmpty();
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/repository/ApplicantRepository.java:46]: suggestion (non-blocking) and praise: This is really clean, would you mind updating `lookupApplicant` (async) to match?

**Review by carolinedanzi — APPROVED** [review]
Looks nice!

File: universal-application-tool-0.0.1/app/repository/ApplicantRepository.java (hunk 1 above)
Comment (natsid) [reply universal-application-tool-0.0.1/app/repository/ApplicantRepository.java:46]: Oh yeah - good catch! Done.

Comment (natsid) [reply universal-application-tool-0.0.1/app/repository/ApplicantRepository.java:45]: Made the same update to ProgramRepository and QuestionRepository as well.

**Pull request merged**